ui-v2/app/utils/acls-status.js

```diff
@@ -28,6 +28,8 @@ export default function(isValidServerError, P = Promise) {
               if (isValidServerError(e)) {
                 enable(true);
                 authorize(false);
+              } else {
+                return P.reject(e);
```

Comment:
I also spotted this here. So even if I get a 500, that isn't the special 'valid as its the legacy consul' error, reject as normal (which will give you the 500 error page)